encrypted-media-respec.html

```diff
@@ -2089,34 +2078,48 @@
           </ol>
         </section>
 
-        <section id="session-close">
-          <h4>Session Close</h4>
-          <p>The Session Close algorithm is run when the CDM closes the session associated with a <a>MediaKeySession</a> object.</p>
+        <section id="close-session">
+          <h4>Close Session</h4>
+          <p>The Close Session algorithm closes the session associated with a <a>MediaKeySession</a> object.</p>
           <p>Closing a session means that the license(s) and key(s) associated with it are no longer available to decrypt <a def-id="media-data"></a>.
             All <a>MediaKeySession</a> methods will fail for this object.
           </p>
-          <p class="note">The CDM may close a session at any point, such as in response to a <a def-id="close"></a> call, when the session is no longer needed, or when system resources are lost.
-          Keys in other sessions SHOULD be unaffected, even if they have overlapping key IDs.
+          <p>The following steps are run:</p>
+          <ol>
+            <li><p>Let <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
+            <li><p>Let <var>cdm</var> be the CDM instance represented by <var>session</var>'s <var>cdm instance</var> value.</p></li>
+            <li>
+              <p>Use <var>cdm</var> to close the <a href="#key-session">session</a> associated with <var>session</var>.</p>
+              <p class="note">Closing the key session results in the destruction of any license(s) and key(s) that have not been explicitly stored.</p>
```

Comment:
Per the discussion above, "Closing the session" is either identical to or perhaps a superset of "Destroy all licenses and keys and session data that has not been explicitly stored". We should certainly specify that somewhere (I'll do a sweep and ensure we do) and then use the "Close the session" phrase consistently.